docs/topics/native/native-objc-interop.md

```diff
@@ -329,38 +348,39 @@ All instances of `NSMutableString` are copied when passed to Kotlin.
 
 ### Collections
 
-Kotlin collections are converted to Swift/Objective-C collections as described
-in the table above. Swift/Objective-C collections are mapped to Kotlin in the same way,
-except for `NSMutableSet` and `NSMutableDictionary`. `NSMutableSet` isn't converted to
-a Kotlin `MutableSet`. To pass an object for Kotlin `MutableSet`,
-you can create this kind of Kotlin collection explicitly by either creating it 
-in Kotlin with e.g. `mutableSetOf()`, or using the `KotlinMutableSet` class in Swift 
-(or `${prefix}MutableSet` in Objective-C, where `prefix` is the framework names prefix).
+Kotlin collections are converted to Swift/Objective-C collections as described in the [table above](#mappings).
+Swift/Objective-C collections are mapped to Kotlin in the same way, except for `NSMutableSet` and `NSMutableDictionary`.
+
+`NSMutableSet` isn't converted to a Kotlin `MutableSet`. To pass an object for Kotlin `MutableSet`, you can create this
```

Comment:
This is a dense paragraph, mostly one huge sentence, with "create by creating" and two "or" constructions. It would be a smaller problem if the interopedia example had an example for `MutableSet`, but it doesn't as far as I can see.

Reply:
I've split and rephrased it a bit, hope it's better now